clang/runtime/dpct-rt/include/util.hpp

```diff
@@ -1025,6 +1025,17 @@ class args_selector<n_nondefault_params, n_default_params, R(Ts...)> {
 #define DPCT_EXPORT
 #endif
 
+/// Cast a "rvalue reference to a temporary object" to an "lvalue reference to
+/// that temporary object".
+/// The returned lvalue reference is available only before the last step in
+/// evaluating the full-expression that contains this function call.
+/// \param [in] temporary_object The rvalue reference to a temporary object.
+/// \returns The lvalue reference to that temporary object.
+template <typename T>
```

Comment:
suggest move this help to lib_common_utils.hpp for library migration only.

Reply:
done